core.yml

```diff
@@ -556,9 +556,9 @@ paths:
       - AuthLinks
       summary: Create an auth_link
       description: |-
-        Create a new auth_link object by making a POST request to the auth_link endpoint. The new auth_link will effectively delete previous auth-links for that User/applicant, rendering the previous URL(s) invalid. The 'mobile' attribute is optional. If it is specified this number will take preference over the User object mobile number for 2FA SMS verification.
-
-        Returns a created auth_link resource, if the operation succeeded. Returns an error if the post failed (e.g. not supplying required properties).
+        Create a new auth_link object by making a POST request to the auth_link endpoint. The new auth_link will effectively delete previous auth-links for that User/applicant, rendering the previous URL(s) invalid. 
```

Comment:
Inconsistent auth-link and auth_link. No need for User - user is fine.

Reply:
Fixed